UI/src/javascript/api/search-results-client.js

```diff
@@ -0,0 +1,28 @@
+import BaseClient from "./base-client";
+import { getSeoUrlFromId } from "../util";
+
+const getAddShortlistTuitionPartnerUrl = (seoUrl) =>
+  `/search-results?handler=AddShortlistedTuitionPartner&tuitionPartnerSeoUrl=${seoUrl}`;
```

Comment:
For safety it would be good to encode any values used directly in a URL like this using `encodeURIComponent()`.
e.g.:
```suggestion
  `/search-results?handler=AddShortlistedTuitionPartner&tuitionPartnerSeoUrl=${encodeURIComponent(seoUrl)}`;
```